scripts/generate-api.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-dsiable no-console */
+/* eslint-disable vue/sort-keys */
+
+/*
+ * API Importer
+ *
+ * This script imports the API and auto generates methods, types,
+ * interfaces and documentation, given the Swagger JSON output
+ * from the backend. The results of which are then stored in the
+ * path specified by the `output` argument.
+ */
+
+require('dotenv-safe').config()
```

Comment:
I assume it's not possible for us to currently use `import` statments here? Is that due to a limitation of `swagger-typescript-api`?

Reply:
This is more due to the fact that it's not transposed by babel or whatever, which is also why it's JS and TS. I tried setting it up as a TS script, but `ts-node` was being a pain, and for what it's used for, I think this should be fine as a one off